### ISSUE TITLE
Error in PanelList.render():

### ISSUE THREAD

**Issue by webdevsyd** [ctxhou/react-tabtab#79]
Hi @ctxhou I'm getting an error when I try to put the code from this link https://github.com/ctxhou/react-tabtab/blob/master/docs/components/Basic.js to my project component. I'm currently using `"react": "^15.4.0",` as per you readme.md your component supports react v15.

![screen shot 2018-03-14 at 8 12 12 pm](https://user-images.githubusercontent.com/15810023/37401834-64086044-27c4-11e8-924f-6be823e895f6.png)

**Comment by ctxhou** [comment]
Hi @webdevsyd,
Thanks for your report.
I will figure out what happened recently.

**Comment by webdevsyd** [comment]
Thank you @ctxhou hope you can fix it so I can use your component

**Comment by ctxhou** [comment]
@webdevsyd I've fixed it in #81 pr.

You can try it again.

**Comment by webdevsyd** [comment]
Hey @ctxhou I installed v1.3.1

if `<Panel>` is more than 1, same error as above is displayed
```
<Tabs>
            <TabList>
              <Tab>Tab1</Tab>
            </TabList>
            <PanelList>
              <Panel>Content1</Panel>
              <Panel>Content1</Panel>
            </PanelList>
          </Tabs>
```
If the `<Panel>` is only 1, it displays this error `children.map is not a function`

**Comment by ctxhou** [comment]
Hi @webdevsyd 

Really thanks for your report.

There are some issues I didn't notice of react v15.

I've fixed it and published in `v1.4.0`.

Please try it again, thanks.
